packages/electron-updater/src/AppUpdater.ts

```diff
@@ -248,11 +248,19 @@ function createClient(data: string | PublishConfiguration | BintrayOptions | Git
   switch (provider) {
     case "github":
       return new GitHubProvider(<GithubOptions>data)
+    case "s3":
     case "generic":
+      if (provider === "s3") {
+        let s3Data = {
```

Comment:
Is there a way to do this in the previous `case`?

Was giving me a `fall through`, is there a way to say that was deliberate?